.github/workflows/figma-code-connect-check.yml

```
@@ -0,0 +1,66 @@
+name: "Figma Code Connect Dry Run"
+
+on:
+  pull_request:
+    paths: ['**/*.figma.tsx']
+
+jobs:
+  figma-dry-run:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "lts/*"
+          cache: "npm"
+
+      - name: Install dependencies
+        run: npm ci
+      
+      - name: Figma Dry Run
```

[imo]

他のStepと命名規則を統一するならこっちのほうが良さそう
```suggestion
      - name: Dry Run Publish Code Connect
```

---

ありがとうございます！